FirebaseInstallations/Source/Library/InstallationsIDController/FIRInstallationsIDController.m

```diff
@@ -456,4 +459,23 @@ - (BOOL)isDefaultApp {
   return [self.appName isEqualToString:kFIRDefaultAppName];
 }
 
+#pragma mark - Keychain
+
++ (NSString *)keychainServiceWithBundleID:(nullable NSString *)bundleID {
+#if TARGET_OS_MACCATALYST || TARGET_OS_OSX
+  // We need to keep service name unique per application on macOS if bundleID provided.
+  // Applications on macOS may request access to Keychain items stored by other applications. It
+  // means that when the app looks up for a relevant Keychain item in the service scope it will
+  // request user password to grant access to the Keychain if there are other Keychain items from
+  // other applications stored under the same Keychain Service.
```

Comment:
This comment is not helpful for me, a non Mac developer. Maybe we can rewrite it a bit?
a) What is "service name"?
b) Why does the "server name" only have to be unique if a Bundle ID was provided?
c) What is "relevant Keychain item"?
d) If an app requests access to another app's data, Mac asks for a user password - do I understand that correctly?
e) Is requesting a user password the use case that we want to prevent?

Reply:
Discussed offline.
@andirayo Please let me know if you still have questions. 

As for Bundle ID - as we discussed, I decided to use Firebase App ID instead.